src/OSPSuite.Core/Domain/EntityExtensions.cs

```diff
@@ -12,6 +12,19 @@ public static class EntityExtensions
       }
 
       public static string EntityPath(this IEntity entity)
+      {
+         return entityPathAsListFor(entity).ToPathString();
+      }
+
+      public static string ConsolidatedPath(this IEntity entity)
```

Comment:
Being able to create a path without the first entry is a requirement as it is used in many scenarios within a simulation.